tests/unit_tests/signature.cpp

```diff
@@ -45,13 +45,13 @@ namespace {
         CryptoPP::RSA::PrivateKey private_key;
         private_key.GenerateRandomWithKeySize(prng, 2048);
 
-        pgp::multiprecision_integer n = private_key.GetModulus();
-        pgp::multiprecision_integer e = private_key.GetPublicExponent();
+        pgp::multiprecision_integer n{ private_key.GetModulus()         };
```

Comment:
Is it really better to align the closing brackets? It just looks weird to me.

Reply:
I agree, but this is a style that we use in many places (e.g. `wallet-proxy`), so I just automatically did it.
@omartijn, is this a style we want in this library as well, or shall I remove the extra padding?

Reply:
It's the style I always like to use, but we don't have any clear style-guide on this. Perhaps we should define it by writing a style document for it?

Reply:
I think that would be a great idea. A quick search suggested that the Core Guidelines, Google, and LLVM styles are somewhat popular.

I'm not overly fond of the Google style guide, but the other two look reasonable, with the notable exception that none of them like `#pragma once`.

Reply:
I am in favor of picking a style guide and adding it as part of the checks. I would argue that this should be an existing guide with preferably no "tweaks" of our own. I would also argue that this should be handled in a separate PR. Opinions?

BTW, `clang-format` (and consequently `clang-tidy`) supports [these](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#configurable-format-style-options) style options.

Reply:
I don't think a PR is the right way to do this (at least for now) since this would affect all Summitto code, not just one repo. So yes, we should leave it out of this one.